### PR TITLE
✨ Add RTV serve_mode

### PR DESCRIPTION
**Pull request by jonathantyng-amp**
Add a `serve_mode` that uses a build from a specific RTV branch. Set the `serve_mode` with the RTV number you want (including the first 2 version digits).

E.g.
`http://localhost:8000/serve_mode=001907231757120` will set the serve mode to use the `001907231757120` RTV branch and requests will go to `https://cdn.ampproject.org/rtv/001907231757120/*`

### CODE REVIEW

File: build-system/app-utils.js [hunk 1]
```diff
@@ -57,34 +103,8 @@ const replaceUrls = (mode, file, hostName, inabox, storyV1) => {
...
-      /\/dist\/v0\/examples\/(.*)\.max.js/g,
-      '/dist/v0/examples/$1.js'
-    );
-    file = file.replace(
-      /\/dist.3p\/current\/(.*)\.max.html/g,
-      hostName + '/dist.3p/current-min/$1.html'
-    );
+    file = replaceCompiledUrls(file, hostName);
+
     if (inabox) {
```
Comment (jonathantyng-amp) [inline build-system/app-utils.js]: @lannka are there any `inabox` URL replacements that need to be done to use RTV urls?

Comment (lannka) [reply build-system/app-utils.js]: thanks for asking. `amp4ads-v0.js` needs to be versioned.
amp4ads-lite-v0.js is obsolute, pls just remove

**Comment by jonathantyng-amp** [comment]
cc @zhouyx for review :)

**Comment by zhouyx** [comment]
LGTM! Any idea on if the rtv doesn't exist, should we just let it timeout?

**Comment by jonathantyng-amp** [comment]
> LGTM! Any idea on if the rtv doesn't exist, should we just let it timeout?

Yeah I think thats the best option

**Comment by zhouyx** [comment]
Just used this feature and it's extremely useful!! THANK YOU 🎉

**Pull request merged**